src/util.js

```diff
@@ -110,7 +110,7 @@ export function format_memory_and_limit(usage, limit) {
 
 // TODO: handle different kinds of errors
 function handleVarlinkCallError(ex) {
-    console.error("Failed to do varlinkcall:", JSON.stringify(ex));
+    console.error("Failed to do varlinkcall:", ex);
```

Comment:
This is not good change for too reasons:
1rst when you are running the tests, and you are seeing the javascript logs in your terminal, you need JSON.stringify to see the actual object.
2nd this change is not relevant to this PR, it's good approach to keep the PRs do exactly what they mention in the commit message, so that in case something break, it will be easier to find which commit broke it from the history.

Reply:
thanks, I will change it back.